publishers/romeo.py

```diff
@@ -79,7 +77,7 @@ def perform_romeo_query(self, search_terms):
         # Parse it
         try:
             parser = ET.XMLParser(encoding='ISO-8859-1')
-            root = ET.parse(BytesIO(response), parser)
+            root = ET.parse(BytesIO(req.content), parser)
```

Comment:
What about something like `root = ET.fromstring(req.text)`? This would let `requests` do the encoding handling on its own, maybe safer?

Reply:
It looks like it works indeed! I think this approach was inherited from the days of urlopen…

Reply:
Actually no:
`E   ValueError: Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration.`

Reply:
So I think we can leave it like that - SHERPA declares its encoding in the XML so we can leave the encoding handling to lxml.

Reply:
Sure